docs/xls-33d-multi-purpose-tokens/index.md

```diff
@@ -0,0 +1,13 @@
+---
+html: index.html
+parent: tokens.html
+blurb: Multi-purpose tokens offer a more compact, flexible token type than trust lines.
+labels:
+  - Tokens
+status: not_enabled
+---
+# Multi-purpose Tokens
+
+Multi-purpose tokens (MPTs) are a more compact and flexible type of fungible token.
+
+Unlike trust lines, MPTs do not represent bidirectional debt relationships. Instead, MPTs function more like a unidirectional trust line with only one balance. This reduces the overhead to support common tokenization requirements, including non-monetery use cases such as tracking reputation points in an online game.
```

Comment:
I think you can just remove this index page and use the multi-purpose-tokens.md file in its place.